pkgs/development/tools/pnpm/fetch-deps/pnpm-config-hook.sh

```diff
@@ -23,10 +23,13 @@ pnpmConfigHook() {
     pnpm config set store-dir "$STORE_PATH"
 
     echo "Installing dependencies"
-
-    if [[ -n "$pnpmWorkspace" ]]; then
```

Comment:
Hmm maybe `exit 2` or something if `pnpmWorkspace` (singular, not plural) is used?

Reply:
There's an assert to guard for it in the fetchdeps default.nix, so pnpmWorkspace should never reach the script.

Reply:
> There's an assert to guard for it in the fetchdeps default.nix, so pnpmWorkspace should never reach the script.

If I understand correctly, that assertion is only done for the `pnpmDeps` (fixed-output) derivation, and not the main derivation that uses these `pnpmDeps`.

Reply:
Fair enough, I'll add the check.